src/Uno.UI/UI/Xaml/Controls/CommandBar/CommandBarFlyout.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Windows.UI.Xaml.Controls
+{
+	partial class CommandBarFlyout
+	{
+		protected override Control CreatePresenter() => throw new NotImplementedException();
```

Comment:
Add [Notimplemented] attribute

Reply:
That's not something public.

Reply:
should this be a `NotSupportedException`?

Reply:
It's not implemented...  the difference between NotImplemented and NotSupported could be subtile:
* Not Implemented: not done yet
* Not Supported: a limitation is preventing its implementation